src/tls.c

```diff
@@ -3042,12 +3061,10 @@ static int TLSX_CSR_Parse(WOLFSSL* ssl, byte* input, word16 length,
                 if (offset + resp_length != length)
                     ret = BUFFER_ERROR;
             }
-        #if !defined(NO_WOLFSSL_SERVER)
```

Comment:
This code isn't server specific. Client needs to be able to verify certificate status, too. See: https://www.wolfssl.com/forums/topic1639-example-of-a-client-that-doesnt-attempt-to-contact-ocsp-responder.html

Reply:
Already checking !NO_WOLFSSL_CLIENT!
Change is good.